src/applications/gi/reducers/calculator.js

```diff
@@ -10,7 +10,7 @@ import {
   FETCH_PROFILE_SUCCEEDED,
 } from '../actions';
 
-const beneficiaryZIPRegExTester = /\b\d{1,5}\b/;
+const beneficiaryZIPRegExTester = /^\d{1,5}$|^\d{1,5}-\d{4}$/;
```

Comment:
Pardon the new guy budding in. This regex will match zipcodes less than 5 digits long. If it's not validating while the user is typing, then it should only match 5 digits.